README.md

```diff
@@ -2,77 +2,31 @@
-
-The Assignee is responsible for determining when no serious objections remain. In particular, there is no need to call for a formal vote on the resolution. If no explicit objections have been raised within, typically, 72 hours for “ordinary” issues and 1 week for “major” issues, the Assignee should assume that there are none. This is known as “lazy consensus.” When this state has been reached, the Assignee is responsible for ensuring that the final consensus has been recorded in the RFC issue before closing it and proceeding with implementation of the decision.
-
-The requestor must be especially careful about not making irreversible changes in the “lazy consensus” time period unless they are absolutely certain there’s a general agreement on the stated course of action. If something is broken, the requestor must be ready to fix it. It is critical to apply sound reasoning and good judgment about what may be acceptable and what might be not. Mistakes will happen; accept that occasionally there will be a requirement to revert an action for which it was thought agreement existed.
 
 **How to start a RFC**
-* Create a [new RFC Issue](https://github.com/heather999/RequestForComments/issues) making sure to provide an adequate description of the situation and proposal as well as a "Planned End" date for comments.
-* If you will be responsible for implementing the proposal, assign the issue to yourself.  If you have reached agreement with someone else to take responsibility, assign them.  If it is unclear who will handle the implementation, leave the issue unassigned and the DESC Computing Change Control Board (DCCCB) will identify an assignee.
+* Create a [new RFC Issue](https://github.com/LSSTDESC/RequestForComments/issues) making sure to provide an adequate description of the situation and proposal.
 * Once the issue is created, a notification will be posted to the #desc-rfc Slack channel
-* All are welcome to comment through the indicated "Planned End" date.
-
-**DESC Computing Change Control Board**
-
-The DESC Computing Change Control Board (DCCCB) responsibilties include:
-* Review all RFCs 
-* Identify an assignee for unassigned RFCs
-* Consider all "escalated" RFCs for approval or withdrawal.
+* All are welcome to comment.
 
-Membership:  
-* Computing Coordinator
-* Deputy Computing Coordinator 
-* Computing Working Group Conveners
-* Computing Deputy Operations Manager
-
-**Escalate a RFC**
-
-Anyone, including DCCB members, may "escalate" an RFC if the issue requires an RFC or if the dicussion is failing to reach a consensus.  Escalated RFCs will be considered by the DCCCB for approval or withdrawal. 
 
 **Adoption**
 
 Before an RFC may be adopted, one or more GitHub issues must be created to capture the work required to implement the RFC’s decision.
```

Comment:
maybe "will likely" instead of "must be" : just to avoid showing more determination than we actually have at that stage about the process?

Reply:
changed to "should be"